lm_human_preference_details/train_policy_accelerate.py

```diff
@@ -510,6 +514,37 @@ def train(args: Args):
     )
     dataloader = DataLoader(dataset, batch_size=args.ppo.local_batch_size)
     policy, optimizer, dataloader = accelerator.prepare(policy, optimizer, dataloader)
+    if args.deepspeed:
+        import deepspeed
+
+        deepspeed_states = AcceleratorState().deepspeed_plugin
+        deepspeed_states.deepspeed_config['train_micro_batch_size_per_gpu'] = args.ppo.local_micro_batch_size
+        deepspeed_states.deepspeed_config['checkpoint'] = {'use_node_local_storage': True}
+        off_load_device = "cpu"
+        stage = 3
+        eval_ds_config = {
+            "train_micro_batch_size_per_gpu": deepspeed_states.deepspeed_config['train_micro_batch_size_per_gpu'],
+            "steps_per_print": 10,
+            # "zero_optimization": {
+            #     "stage": stage,
+            #     "stage3_param_persistence_threshold": 1e4,
+            #     "offload_param": {
+            #         "device": off_load_device
+            #     }
+            # },
+            "bf16": {
+                "enabled": True
+            },
+            "prescale_gradients": False,
```

Comment:
I think this flag and the one below are false by default, so probably don't need to be set either